migration/src/main/java/com/intershop/customization/migration/file/MoveFolder.java

```diff
@@ -57,15 +61,85 @@ public void migrate(Path cartridgeDir)
                 if (!targetPath.toFile().exists())
                 {
                     Files.move(sourcePath, targetPath);
+                    context.recordSuccess(cartridgeName, MigrationContext.OperationType.MOVE, sourcePath, targetPath);
                 }
                 else {
                     LoggerFactory.getLogger(getClass()).warn("Folder '{}' exists.", targetPath);
+                    context.recordSkipped(cartridgeName, MigrationContext.OperationType.MOVE, sourcePath, targetPath,
+                            "Target folder already exists");
                 }
             }
             catch(IOException e)
             {
+                context.recordFailure(cartridgeName, MigrationContext.OperationType.MOVE, sourcePath, targetPath,
+                        e.getMessage());
                 throw new RuntimeException(e);
             }
         }
+
+        checkRemainingStaticFiles(cartridgeDir, cartridgeName, context);
+    }
+
+    /**
+     * Checks if the staticfiles directory still exists after migration and contains unmapped directories. Records any
+     * remaining directories as unknown operations.
+     *
+     * @param cartridgeDir The cartridge directory
+     * @param cartridgeName The name of the cartridge
+     * @param context The migration context
+     */
+    private void checkRemainingStaticFiles(Path cartridgeDir, String cartridgeName, MigrationContext context)
+    {
+        Path staticFilesDir = cartridgeDir.resolve("staticfiles");
+        if (Files.exists(staticFilesDir))
+        {
+            try
+            {
+                // Check if staticfiles/cartridge exists and has content
+                Path cartridgeStaticDir = staticFilesDir.resolve("cartridge");
+                if (Files.exists(cartridgeStaticDir))
+                {
+                    try (var paths = Files.list(cartridgeStaticDir))
+                    {
+                        paths.filter(Files::isDirectory).forEach(dir -> {
+                            Path relativePath = cartridgeDir.relativize(dir);
+                            LOGGER.warn("Unmapped directory found in staticfiles/cartridge: {}", relativePath);
+                            context.recordUnknown(cartridgeName, MigrationContext.OperationType.MOVE, dir, null,
+                                    "Unmapped directory in staticfiles/cartridge");
+                        });
+                    }
+                }
```

Comment:
Please rebase your branch and check which parts you can replace by using the new `FileUtils` class.